src/main/java/org/jabref/logic/ai/AiDefaultPreferences.java

```diff
@@ -76,6 +77,45 @@ public class AiDefaultPreferences {
 
     public static final int CONTEXT_WINDOW_SIZE = 8196;
 
+    public static final Map<AiTemplate, String> TEMPLATES = Map.of(
+            AiTemplate.CHATTING_SYSTEM_MESSAGE,
+                    "You are an AI assistant that analyses research papers. You answer questions about papers.\n" +
+                            "You will be supplied with the necessary information. The supplied information will contain mentions of papers in form '@citationKey'.\n" +
+                            "Whenever you refer to a paper, use its citation key in the same form with @ symbol. Whenever you find relevant information, always use the citation key.\n\n" +
+                            "Here are the papers you are analyzing:\n" +
+                            "#foreach( $entry in $entries )\n" +
+                            "${CanonicalBibEntry.getCanonicalRepresentation($entry)}\n" +
+                            "#end",
+
+            AiTemplate.CHATTING_USER_MESSAGE, """
+                    $message
+
+                    Here is some relevant information for you:
+                    #foreach( $excerpt in $excerpts )
+                    ${excerpt.citationKey()}:
+                    ${excerpt.text()}
+                    #end""",
+
+            AiTemplate.SUMMARIZATION_CHUNK, """
+                Please provide an overview of the following text. It's a part of a scientific paper.
+                The summary should include the main objectives, methodologies used, key findings, and conclusions.
+                Mention any significant experiments, data, or discussions presented in the paper.
+
+                DOCUMENT:
+                $document
+
+                OVERVIEW:""",
+
+            AiTemplate.SUMMARIZATION_COMBINE, """
+                You have written an overview of a scientific paper. You have been collecting notes from various parts
+                of the paper. Now your task is to combine all of the notes in one structured message.
+
+                SUMMARIES:
+                $summaries
+
+                FINAL OVERVIEW:"""
```

Comment:
I don't understand that - the FINAL OVERVIEW should be generated? Why is it part of the prompt?